File: Assignment/cornflakes/roadtax.py
priceofcar = int (input("Enter price of your car: "))

if priceofcar < 1_000_000:
	tax = priceofcar / 100
	roadtax = tax * 10
	print(f'Your road tax is {roadtax}')
    
elif priceofcar >= 1_000_000 and  3_000_000:
	tax = priceofcar / 100
	roadtax = tax * 15
	print(f'Your road tax is {roadtax}')
    
elif priceofcar > 3_000_000 and 5_000_000:
	tax = priceofcar / 100
	roadtax = tax * 20
	print(f'Your road tax is {roadtax}')
     
elif priceofcar > 5_000_000:
	tax = priceofcar / 100
	roadtax = tax * 25
	print(f'Your road tax is {roadtax}')

    

File: PycharmProjects/jaffa/Cornflakes/RoadTax.java
import java.util.Scanner;
public class RoadTax{
public static void main(String...args){

	Scanner scanner = new Scanner(System.in);
	double roadtax = 0;

	System.out.print("Enter the price of your car: ");
	double price = scanner.nextDouble();

	if (price <= 1000000){
		roadtax = price * 0.10;
		System.out.print(roadtax);
		}
	else if (price >= 1000000 && price < 3000000){
		roadtax = price * 0.15;
		System.out.print(roadtax);
		}
	else if (price >= 3_000_000 && price < 5_000_000){
		roadtax = price * 0.20;
		System.out.print(roadtax);
		}
	else
		roadtax = price * 0.25;
		System.out.print(roadtax);
	}


}